CONTRIBUTING.md

```diff
@@ -114,6 +114,8 @@ the VCS/git logs more valuable. The structure of a commit message is:
   BREAKING CHANGE: refactor to use Python 3 features since Python 2 is no longer supported.
   ```
 
+Use `make lintcommit` locally to test if your PR passes the commit message tests.
```

Comment:
Let's mention this parenthetically in the first sentence (line 85):

```
Follow the [conventional commits guidelines][conventional_commits] to *make reviews easier* and to make
the VCS/git logs more valuable. (Try `make lintcommit`.) The structure of a commit message is:
```

It is very easy to miss these "dangling notes" after the main part of a section.